apps/backend/src/libs/plugins/authorization-plugin/authorization-plugin.ts

```diff
@@ -39,7 +39,7 @@ const authorizationPlugin = fp<PluginOptions>((app, options, done) => {
 				payload: { exp, userId },
 			} = await token.decode(headerToken);
 
-			if ((exp as number) < Date.now()) {
+			if ((exp as number) < new Date().getSeconds()) {
```

Comment:
This is not what you need, check the docs
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getSeconds